Caboodle/DeviceInfo/DeviceInfo.shared.cs

```diff
@@ -98,8 +98,22 @@ public enum DeviceType
         Virtual
     }
 
-    public class ScreenMetrics
+#if __ANDROID__
+    [Android.Runtime.Preserve(AllMembers = true)]
+#elif __IOS__
+    [Foundation.Preserve(AllMembers = true)]
+#endif
+    public struct ScreenMetrics
```

Comment:
Do we have any idea why this type gets removed if the device info is being used?

Reply:
I don't to be honest with you. It is kind of odd because other ones are just fine. I would have contact the linker team i guess. However not too worried about a single struct.

Reply:
Maybe it is because it is a struct... the other types are classes.

Reply:
it was a class originally, i changed to struct :)

Reply:
:sob: